pkgs/top-level/all-packages.nix

```diff
@@ -16106,11 +16106,18 @@ with pkgs;
     fpc = fpc;
   };
 
-  lazarus-qt = libsForQt5.callPackage ../development/compilers/fpc/lazarus.nix {
```

Comment:
An alias for `lazarus-qt` should be added to `pkgs/top-level/aliases.nix` for backwards compatibility